_co-op-companies/N-Ubisoft.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Ubisoft"
+identification: "ubisoft"
+description: "Ubisoft is a entertainment software company."
+link: "https://winnipeg.ubisoft.com/"
+image: "assets/img/logos/Ubisoft.png"
+width: "200px"
+complete: true
+members:
+  -name: "Raquel"
+  summary: "used C#, .net, vue, on work term 2 in Fall 2019"
```

Comment:
Raquel's information isn't displaying properly because the members variables have been updated in master so summary no longer exists. The new attributes include term, tools, and image.

Reply:
Changed summery to tools, term, and image.